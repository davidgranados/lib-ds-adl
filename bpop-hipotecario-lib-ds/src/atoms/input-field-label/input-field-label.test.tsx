import React from "react";
import { mount, ReactWrapper } from "enzyme";
import InputFieldLabel from "./input-field-label";

describe("<InputFieldLabel />", () => {
  let wrapper: ReactWrapper<typeof InputFieldLabel>;
  beforeEach(() => {
    jest.clearAllMocks();
    // given
    wrapper = mount<typeof InputFieldLabel>(<InputFieldLabel>label</InputFieldLabel>);
  });
  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
