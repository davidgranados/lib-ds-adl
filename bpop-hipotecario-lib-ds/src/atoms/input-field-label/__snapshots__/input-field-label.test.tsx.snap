// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputFieldLabel /> should match snapshot 1`] = `
<label
  className="default"
>
  label
</label>
`;
